setup.py

```diff
@@ -1,28 +1,26 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
-try:
```

Comment:
Let this try/except stay?

Reply:
What is the point of the try/except?

Reply:
Messy permissions on some system can cause this to fail in hilarious ways. It doesn't matter much though.

I'm gonna switch this package to using `flit` for packaging if I get the time.

Reply:
I think you need to fix your permissions. Writing crappy software because someone doesn't have their system set up rioght is usually not a good idea. But given that you are the maintainer here, you are welcome to make the change yourself if you really think it is a good idea.